core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aas/DefaultPropertyMapper.java

```diff
@@ -23,13 +23,17 @@
 public class DefaultPropertyMapper implements PropertyMapper<Property> {
    @Override
    public Property mapToAasProperty( final Type type, final org.eclipse.esmf.metamodel.Property property, final Context context ) {
-      return new DefaultProperty.Builder()
-            .idShort( context.getPropertyShortId() )
-            .valueType( AasDataTypeMapper.mapAspectTypeToAasXsdDataType( mapType( type ) ) )
-            .displayName( LangStringMapper.NAME.map( property.getPreferredNames() ) )
-            .value( context.getPropertyValue( UNKNOWN_EXAMPLE ) )
-            .semanticId( buildReferenceToGlobalReference( property ) )
-            .build();
+      final DefaultProperty defaultProperty = new DefaultProperty();
+      defaultProperty.setIdShort( context.getPropertyShortId() );
+      defaultProperty.setValueType( AasDataTypeMapper.mapAspectTypeToAasXsdDataType( mapType( type ) ) );
+      defaultProperty.setDisplayName( LangStringMapper.NAME.map( property.getPreferredNames() ) );
+      defaultProperty.setSemanticId( buildPropertyReferenceToGlobalReference( property ) );
```

Comment:
Why propertyReference?

Reply:
Could you please explain more, because in this method "buildPropertyReferenceToGlobalReference" we building GlobalReference, not property reference?

@BirgitBoss